src/watertap_contrib/seto/unit_models/surrogate/lt_med_surrogate.py

```diff
@@ -113,7 +113,7 @@ class LTMEDData(UnitModelBlockData):
         "number_effects",
         ConfigValue(
             default=12,
-            domain=In([i for i in range(3, 15)]),
+            domain=PositiveInt,
```

Comment:
Would this allow for number effects >15 or <3?

Reply:
Yes. We would like to raise a more explicit error message when inputting invalid values. The imbedded `ValueError `from ConfigBlock is a big Vague, by just saying "invalid value for configuration 'number_effects'".

The more straightforward message was added at line 129.